### PR TITLE
Fix header counter

### PR DESCRIPTION
**Pull request by marcoambrosini**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: templates/header-top-navbar.php [hunk 1]
```diff
@@ -1,53 +1,99 @@
 <?php $l = new L10N('header-navbar');
 require get_template_directory() . '/strings.php';
 ?>
-<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"> -->
+<!-- <meta http-equiv="refresh" content="60"> -->
```
Comment (juliushaertl) [inline templates/header-top-navbar.php:4]: :fire:

**Review by juliushaertl — APPROVED** [review]
Seemed to work :+1:

File: templates/header-top-navbar.php (hunk 1 above)
Comment (jospoortvliet) [reply templates/header-top-navbar.php:4]: why did THAT happen? The refresh especially isn't nice ;-)

**Review by jospoortvliet — APPROVED** [review]
I can't speak on the rest but if Julius says it's good, it must be. Will test it at home and if ok, deploy.

**Pull request merged**